### PR TITLE
Display an X next to disabled breakpoints

### PR DESCRIPTION
**Pull request by asmeurer**
I also plan on adding a theme color for this.  Let me know how you feel about the `X` and the `# hits` UI (I'm happy to change/remove either).

### CODE REVIEW

**Comment by asmeurer** [comment]
This is ready to be reviewed.

**Comment by inducer** [comment]
Looks good, feel free to merge.

File: pudb/debugger.py [hunk 1]
```diff
@@ -546,6 +546,8 @@ def get_lines(self, debugger_ui):
             return [SourceLine(debugger_ui, self.file_name)]
 
         breakpoints = debugger_ui.debugger.get_file_breaks(self.file_name)[:]
+        breakpoints = [lineno for lineno in breakpoints if
```
Comment (asmeurer) [inline pudb/debugger.py]: I think this line is unnecessary. It might be necessary if we ever save the "enabled" state to the breakpoints file, however.

**Comment by asmeurer** [comment]
Would love to hear folks' opinions on my choice of theme colors for disabled breakpoints. I am happy with the "midnight" colors, but that's the only theme I use. The rest I tried to find something that looks OK, but if you use the theme and want them to look better, let me know.

**Comment by inducer** [comment]
I'll change them once they bother me. :)

**Pull request merged**